<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionTypesSelectorDialog</class>
 <widget class="QDialog" name="ActionTypesSelectorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выберите действия</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblMesInfo">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="pnlWidgets" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lblSelectedCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Количество выбранных</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblFindByCode">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>| Поиск по</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSearchMode">
        <property name="text">
         <string>наименованию</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtFindByCode">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="pnlSelectedActionTypes" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabSelected">
          <attribute name="title">
           <string>Назначить</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="CCheckedActionsTableView" name="tblSelectedActionTypes"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabExists">
          <attribute name="title">
           <string>Назначено</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="CTableView" name="tblExistsClientActions"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pnlActionTypes" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <widget class="CActionTypeAnalysesTreeView" name="treeActionTypeGroups"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QStackedWidget" name="stackedFilters">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageFilters">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QWidget" name="pnlFilter" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <property name="horizontalSpacing">
           <number>6</number>
          </property>
          <item row="3" column="4" colspan="2">
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="chkNecessity">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>ЧП</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="edtNecessity">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximum">
                <double>1.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="btnAmountFromMES">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Количество из МЭС</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="chkContract">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&amp;Договор</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QCheckBox" name="chkPreferable">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>П&amp;редпочтение</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="chkOnlyNotExists">
            <property name="text">
             <string>Отсутствующие</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="chkIndications">
            <property name="text">
             <string>По показаниям</string>
            </property>
           </widget>
          </item>
          <item row="2" column="8">
           <widget class="QCheckBox" name="chkOrgStructure">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>П&amp;одразделение</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QCheckBox" name="chkUseDates">
            <property name="text">
             <string>Даты</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QCheckBox" name="chkSexAndAge">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&amp;Пол и возраст</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QCheckBox" name="chkKsg">
            <property name="text">
             <string>КСГ</string>
            </property>
           </widget>
          </item>
          <item row="3" column="6">
           <widget class="QCheckBox" name="chkMedicament">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Медикаменты</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QCheckBox" name="chkMes">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&amp;МЭС</string>
            </property>
           </widget>
          </item>
          <item row="3" column="9">
           <widget class="CRBComboBox" name="cmbSpeciality">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="chkNomenclative">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&amp;Номенклатура</string>
            </property>
           </widget>
          </item>
          <item row="3" column="8">
           <widget class="QCheckBox" name="chkPlanner">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>П&amp;ланировщик</string>
            </property>
           </widget>
          </item>
          <item row="2" column="9">
           <widget class="COrgStructureComboBox" name="cmbOrgStructure">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QPushButton" name="btnAllActionTypes">
            <property name="text">
             <string>Все услуги</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="chkMKB">
            <property name="text">
             <string>МКБ</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageFavourites">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QWidget" name="pnlFavourites" native="true">
         <widget class="QPushButton" name="btnFavouriteActionTypes">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>211</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Избранное</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4"/>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes</header>
  </customwidget>
  <customwidget>
   <class>CCheckedActionsTableView</class>
   <extends>QTableView</extends>
   <header>Events/ActionsSelectorSelectedTable.h</header>
  </customwidget>
  <customwidget>
   <class>CActionTypeAnalysesTreeView</class>
   <extends>QTreeView</extends>
   <header>Events/ActionTypeComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>chkOnlyNotExists</tabstop>
  <tabstop>chkContract</tabstop>
  <tabstop>chkOrgStructure</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>chkNomenclative</tabstop>
  <tabstop>chkMes</tabstop>
  <tabstop>chkNecessity</tabstop>
  <tabstop>edtNecessity</tabstop>
  <tabstop>btnAmountFromMES</tabstop>
  <tabstop>chkPlanner</tabstop>
  <tabstop>cmbSpeciality</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>tblSelectedActionTypes</tabstop>
  <tabstop>treeActionTypeGroups</tabstop>
  <tabstop>edtFindByCode</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>tblExistsClientActions</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ActionTypesSelectorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>777</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ActionTypesSelectorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>777</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkNecessity</sender>
   <signal>clicked(bool)</signal>
   <receiver>edtNecessity</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkMes</sender>
   <signal>clicked(bool)</signal>
   <receiver>chkNecessity</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkPlanner</sender>
   <signal>clicked(bool)</signal>
   <receiver>cmbSpeciality</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>698</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>752</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkOrgStructure</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbOrgStructure</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>698</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>752</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkMes</sender>
   <signal>clicked(bool)</signal>
   <receiver>btnAmountFromMES</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkMes</sender>
   <signal>clicked(bool)</signal>
   <receiver>chkMedicament</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
